homeassistant/components/tuya/strings.json

```
@@ -615,7 +615,12 @@
         "name": "Last operation duration"
       },
       "water_level": {
-        "name": "Water level"
+        "name": "Water level",
+        "state": {
+          "level_1": "Low",
+          "level_2": "Medium",
```

You can replace these two with common states:

```suggestion
          "level_1": "[%key:common::state::low%]",
          "level_2": "[%key:common::state::medium%]",
```

I see if we can add common states for "Empty" and "Full" to HA so you could use the latter, too.

---

Thank you! Sounds good!